src/app/ruler/ruler.css

```diff
@@ -122,3 +121,28 @@ body {
 	border-color: black !important;
 	color: black;
 }
+
+#guide__vertical, #guide__horizontal {
+	position: fixed;
+	box-sizing: border-box;
+}
+
+#guide__vertical {
+	top:0;
+	left: 50%;
+	height: 100%;
+	width: 1px;
+	border-right: dashed red 1px;
```

Comment:
Not personally a big fan of the red haha, maybe in white/gray or lighter red? Don't worry about it though, we'll find one haha